tests/test_grids/test_sgrid.py

```diff
@@ -6,6 +6,14 @@
 import xarray_subset_grid.accessor  # noqa: F401
 from xarray_subset_grid.utils import ray_tracing_numpy
 # open dataset as zarr object using fsspec reference file system and xarray
+
+from xarray_subset_grid.utils import get_test_file_dir
+
+from gridded import Variable, VectorVariable
+
```

Comment:
I tend to put this inline:

`TEST_DIR = Path(__file__).parent.parent / 'test_data'`

`sample_sgrid_file = TEST_DIR / 'arakawa_c_test_grid.nc'`